plugins/database/influxdb/influxdb_test.go

```diff
@@ -192,112 +331,104 @@ func TestInfluxdb_RevokeDeletedUser(t *testing.T) {
+	newUserReq := newdbplugin.NewUserRequest{
+		UsernameConfig: newdbplugin.UsernameMetadata{
+			DisplayName: "test",
+			RoleName:    "test",
+		},
+		Statements: newdbplugin.Statements{
+			Commands: []string{createUserStatements},
+		},
+		Password:   initialPassword,
+		Expiration: time.Now().Add(1 * time.Minute),
 	}
+	newUserResp := dbtesting.AssertNewUser(t, db, newUserReq)
 
-	usernameConfig := dbplugin.UsernameConfig{
-		DisplayName: "test",
-		RoleName:    "test",
-	}
-
-	username, password, err := db.CreateUser(context.Background(), statements, usernameConfig, time.Now().Add(time.Minute))
-	if err != nil {
-		t.Fatalf("err: %s", err)
-	}
-
-	config.Username, config.Password = username, password
-	if err = testCredsExist(t, config); err != nil {
-		t.Fatalf("Could not connect with new credentials: %s", err)
-	}
+	assertCredsExist(t, config.URL().String(), newUserResp.Username, initialPassword)
 
 	// call cleanup to remove database
 	cleanup()
 
+	assertCredsDoNotExist(t, config.URL().String(), newUserResp.Username, initialPassword)
+
 	// attempt to revoke the user after database is gone
-	err = db.RevokeUser(context.Background(), statements, username)
+	delReq := newdbplugin.DeleteUserRequest{
+		Username: newUserResp.Username,
+	}
+	_, err := db.DeleteUser(context.Background(), delReq)
 	if err == nil {
 		t.Fatalf("Expected err, got nil")
 	}
+	assertCredsDoNotExist(t, config.URL().String(), newUserResp.Username, initialPassword)
 }
 
 func TestInfluxdb_RevokeUser(t *testing.T) {
 	cleanup, config := prepareInfluxdbTestContainer(t)
 	defer cleanup()
 
 	db := new()
-	_, err := db.Init(context.Background(), config.connectionParams(), true)
-	if err != nil {
-		t.Fatalf("err: %s", err)
+	req := newdbplugin.InitializeRequest{
+		Config:           config.connectionParams(),
+		VerifyConnection: true,
 	}
+	dbtesting.AssertInitialize(t, db, req)
 
-	statements := dbplugin.Statements{
-		Creation: []string{testInfluxRole},
+	initialPassword := "nuozxby98523u89bdfnkjl"
+	newUserReq := newdbplugin.NewUserRequest{
+		UsernameConfig: newdbplugin.UsernameMetadata{
+			DisplayName: "test",
+			RoleName:    "test",
+		},
+		Statements: newdbplugin.Statements{
+			Commands: []string{createUserStatements},
+		},
+		Password:   initialPassword,
+		Expiration: time.Now().Add(1 * time.Minute),
 	}
+	newUserResp := dbtesting.AssertNewUser(t, db, newUserReq)
 
-	usernameConfig := dbplugin.UsernameConfig{
-		DisplayName: "test",
-		RoleName:    "test",
-	}
-
-	username, password, err := db.CreateUser(context.Background(), statements, usernameConfig, time.Now().Add(time.Minute))
-	if err != nil {
-		t.Fatalf("err: %s", err)
-	}
+	assertCredsExist(t, config.URL().String(), newUserResp.Username, initialPassword)
 
-	config.Username, config.Password = username, password
-	if err = testCredsExist(t, config); err != nil {
-		t.Fatalf("Could not connect with new credentials: %s", err)
+	// attempt to revoke the user after database is gone
```

Comment:
I don't think the database is gone at this point?

Reply:
Also see below: after calling `cleanup()` the container is destroyed. However I realized the test can run against an already running Influx instance and then `cleanup()` does nothing and the test would fail. I've updated it to attempt to delete a user that doesn't exist.

Reply:
That makes sense for `TestInfluxdb_RevokeDeletedUser()`, but this is `TestInfluxdb_RevokeUser()`, which I think is just testing the delete user happy path.

Reply:
Oh, you're correct. The comment is copy-pasted and needs to be removed.